llvm/lib/Target/PowerPC/PPCRegisterInfo.td

```diff
@@ -798,6 +798,7 @@ def directbrtarget : Operand<OtherVT> {
 def absdirectbrtarget : Operand<OtherVT> {
   let PrintMethod = "printAbsBranchOperand";
   let EncoderMethod = "getAbsDirectBrEncoding";
+  let DecoderMethod = "decodeDirectBrTarget";
```

Comment:
This seems like a bug related to branch instruction decode?
Normally we need another patch to fix this as the fix here is not same with this patch's purpose.
Since this change is quite small, let's put them together.